### ISSUE TITLE
Carousel text not responsive

### ISSUE THREAD

**Issue by rmporsch** [shamlab/shamlab.github.io#3]
Currently pictures inserted into the carousel are likely misfitted as can be seen with the current ShamLab logo.

**Comment by rmporsch** [comment]
Found a possible solution [here](http://stackoverflow.com/questions/17357306/bootstrap-carousel-resizing-image) which suggests to insert:

```
.carousel-inner > .item > img, .carousel-inner > .item > a > img {
    width: 100%;
}
``` 

into the carousel css file.

**Comment by rmporsch** [comment]
Mh the change (31a38536b10ec8657853302b3e7dea08d2691f54) results in a hugh image. We might need to change some other settings in that file

**Comment by kestrel614** [comment]
Now carousel images are responsive (all the images I am using for testing purposes are 16:9 ratio wallpapers though).

Carousel texts are also responsive except for when you make the window especially narrow, e.g.

![image](https://cloud.githubusercontent.com/assets/3081397/21961732/edd6380c-db4c-11e6-81c3-411d4524c44d.png)

Also the current placeholders are not necessarily free for non-commercial use. I propose to PS something or use simple background tiles (i.e. abstract tiling wallpaper).

**Comment by rmporsch** [comment]
@kestrel614 Did you take that screen shot on your mobile? On the computer the pictures are really large. Too large for my taste.

But the overall problem has been fixed so I am closing this issue

**Comment by kestrel614** [comment]
Yeah but it is still mobile unfriendly so I think I am reopening it with a different name...
